src/pds_doi_service/core/entities/doi.py

```diff
@@ -90,32 +94,35 @@ class DoiEvent(str, Enum):
             Moves a DOI from findable back to registered
 
     """
-    Publish = 'publish'
-    Register = 'register'
-    Hide = 'hide'
+
+    Publish = "publish"
+    Register = "register"
+    Hide = "hide"
 
 
 @dataclass
 class Doi:
     """The dataclass definition for a Doi object."""
+
+    # 🤔 TODO: replace the ``Any`` with what should be in aggregate types:
```

Comment:
I believe in all cases, the `Any`'s can be replaced with `str`

Reply:
Thanks @collinss-jpl! I'm pushing matching updates